site/features.md

```diff
@@ -55,6 +56,28 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="automatic-overlay-selection"></a>Automatic overlay selection
+
+Weave supports two overlay technologies - a custom UDP encapsulation
+('sleeve') featuring optional encryption and underlay PMTU discovery,
+and a VXLAN transport ('fastdp') that leverages the Open vSwitch
+kernel module to provide near-native performance. When connecting to a
+new peer, the router automatically prefers the fastdp overlay under
+the following conditions:
```

Comment:
That's a lot of technical jargon. Overlay? UDP encapsulation? PMTU discovery? VXLAN? Open vSwitch kernel module? ... I don't think a user should have to know about any of these.

How about we call this section "Optimised data path". Explain in a sentence what we mean by "data path". Explain that normally weave will use a "fast in-kernel data path" or s.t. like that.